guides/get-going-with-gitops.mdx

```diff
@@ -0,0 +1,485 @@
+---
+title: Get going with GitOps
+description: Configuring Flipt for a GitOps workflow
+---
+
+## Why GitOps?
+
+GitOps is a set of practices centered around storing your application and infrastructure configuration in Git.
+The goal being to leverage the capabilities of version-control for your entire systems configuration.
+
+If you're already embracing GitOps practices for your other configuration, then aligning your feature flags with these same practices can complete the experience.
+No more correlating what changed in Git, with what changed in your feature flag system to understand the entire state of the world.
+Git is the single source of truth as GitOps intended.
+
+So how do we achieve this with Flipt?
+
+## What We Will Learn
+
+In this guide we will:
```

Comment:
```suggestion
In this guide you will:
```

Think we should go with the second? person perspective to match our other guides: 'you' instead of 'we' where it makes sense.